samples/kind/storage.yaml

```diff
@@ -0,0 +1,91 @@
+apiVersion: ydb.tech/v1alpha1
+kind: Storage
+metadata:
+  name: storage-kind-sample
+spec:
+  monitoring:
```

Comment:
Isn't the disabled monitoring the default?

Reply:
fixed